.github/workflows/ci.yml

```diff
@@ -129,8 +129,8 @@ jobs:
           python -m pip install -r dev_tools/conf/pip-list-dev-tools.txt
           wget https://github.com/bazelbuild/bazel/releases/download/0.26.0/bazel_0.26.0-linux-x86_64.deb
           sudo dpkg -i bazel_0.26.0-linux-x86_64.deb
-      - name: Build protos
-        run: check/build-protos
+      - name: Build changed protos
+        run: check/build-changed-protos
```

Comment:
We're positive we can't break the proto build except by editing the protos themselves? For example, if we update the build-changed-protos file but not the protos themselves...

Reply:
This is a goodpoint.  If the build files are changed, then I think we should build the corresponding proto.

Reply:
PTAL.  Added that it builds all protos if the BUILD rule has changed.